src/libpgmoneta/utils.c

```diff
@@ -1753,6 +1753,21 @@ pgmoneta_copy_directory(char* from, char* to)
    return 1;
 }
 
+static int
+pgmoneta_get_permissions(char* from, int* permissions)
```

Comment:
You need a definition at the top of the file.

And, static functions doesn't have to start with `pgmoneta_` in their name